django_jenkins/management/commands/jenkins.py

```diff
@@ -49,7 +49,25 @@ class Command(TestCommand):
                     help="Module name to exclude"),
         make_option("--project-apps-tests", action="store_true",
                     default=False, dest="project_apps_tests",
-                    help="Take tests only from project apps")
+                    help="Take tests only from project apps"),
+
+        # Required by Django 1.8 if create_parser overrided
```

Comment:
I think it's safe to fix the `create_parser` method rather than adding these options. Currently, it looks like this:

```python
def create_parser(self, prog_name, subcommand):
        # This is what the superclass does.
        test_runner_class = get_runner(settings, self.test_runner)
        options = self.option_list + getattr(
            test_runner_class, 'option_list', ())

        # Here we construct the total options list for the OptionsParser.
        # We could instead use the one returned by the super-ed create_parser,
        # and abuse the `parser.add_option` method to do exactly the same.
        # If there is already an option defined for current text, an exception will be
        # raised so it's safe.
        for task in self.tasks:
            options += tuple(option for option in getattr(task, 'option_list', ())
                             if all(option._long_opts[0] != existing._long_opts[0]
                                    for existing in options))

        return OptionParser(prog=prog_name,
                            usage=self.usage(subcommand),
                            version=self.get_version(),
                            option_list=options)
```

The resulting version would look like this:

```python
def create_parser(self, prog_name, subcommand):
    parser = super(Command, self).create_parser(prog_name, subcommand)

    for task in self.tasks:
        for option in getattr(task, 'option_list', ()):
            try:
                parser.add_option(option)

            except optparse.OptionConflictError:
                continue

    return parser
```